addOns/scanpolicies/src/main/zapHomeFiles/policies/Dev CICD.policy

```diff
@@ -0,0 +1,55 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
+<configuration>
+    <policy>Developer CI/CD</policy>
+    <scanner>
+        <level>OFF</level>
+        <strength>MEDIUM</strength>
+    </scanner>
+    <plugins>
```

Comment:
What sorting is this following? Sorting by ID would make it easier to double check.